custom_components/tahoma/climate_seh.py

```diff
@@ -0,0 +1,68 @@
+"""Support for Atlantic Electrical Heater IO controller."""
+import logging
+from typing import List
+
+from homeassistant.components.climate import ClimateEntity
+from homeassistant.components.climate.const import HVAC_MODE_HEAT, HVAC_MODE_OFF
+from homeassistant.const import TEMP_CELSIUS
+
+from .tahoma_device import TahomaDevice
+
+_LOGGER = logging.getLogger(__name__)
+
+COMMAND_DOWN = "down"
+COMMAND_MY = "my"
+COMMAND_OFF = "off"
+COMMAND_ON = "on"
+COMMAND_UP = "up"
+
+HVAC_MODE_UNKNOWN = "Unknown"
+
+PRESET_UNKNOWN = "Unknown"
+
+
+class StatelessExteriorHeating(TahomaDevice, ClimateEntity):
+    """Representation of TaHoma IO Atlantic Electrical Heater."""
+
+    def __init__(self, tahoma_device, controller):
+        """Init method."""
+        super().__init__(tahoma_device, controller)
+
+    @property
+    def supported_features(self) -> int:
+        """Return the list of supported features."""
+        return 0
```

Comment:
Could we do something with `SUPPORT_FAN_MODE` or `SUPPORT_AUX_HEAT`? Not sure if there are any possibilities to set a preset (up + down). Since this is a RTS device, we will set `assumed_state` to True already, thus I believe both options should be active all the time.

Or using `SUPPORT_PRESET_MODE`?

Reply:
I first started with SUPPORT_PRESET_MODE with PRESET_ECO, PRESET_COMFORT and PRESET_BOOST as the three levels. But there is no way to know what preset is selected, so how many ups and downs should be done to go to the desired preset. One solution could be 2 ups for boost, 2 downs for Eco, and 2 ups 1 down or 2 downs 1 up for comfort. It's a bit overkill, but if it doesn't cycle through the modes, that could work.

Reply:
added presets.